python-api/app/integrations/google_calendar/parse_calendar_api_data.py

```
@@ -0,0 +1,274 @@
+
+    Raises:
+        FileNotFoundError: If the specified file does not exist.
+        ValueError: If the file contains invalid JSON.
+    """
+    try:
+        logger.debug(f"Loading JSON data from {json_path}")
+
+        with open(json_path, "r") as json_file:
+            return json.load(json_file)
+    except FileNotFoundError:
+        raise FileNotFoundError(f"File not found: {json_path}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)